Find a functionally equivalent code:android.content.res.TypedArray.getPositionDescription:COMMENT
Method Modifier: public      
Comment:/**
 * Returns a message about the parser state suitable for printing error messages.
 */

@@ -1,3 +1,6 @@
 {
+    if (mRecycled) {
+        throw new RuntimeException("Cannot make calls to a recycled instance!");
+    }
     return mXml != null ? mXml.getPositionDescription() : "<internal>";
 }

